analyses/mwi/dryspells_trigger/exploration/2021_2022_postseason_overview.Rmd

```diff
@@ -0,0 +1,570 @@
+
+plot(dry_days_per_month_r, 
+     "nov", 
+     main = "Nov",
+     type = "interval", 
+     breaks = dry_days_per_month_breaks, 
+     axes = F,
+     col = monthly_patterns_col,
+     alpha = 0.9,
+     legend = "topleft",
+     plg = list(title = "Dry Days", cex = 0.7),
+     mar = NA,
+     cex.main = 1.3)
+plot(southern_shp, add = T)
+
+plot(dry_days_per_month_r, 
+     "dec", 
+     main = "Dec",
+     type = "interval", 
+     breaks = dry_days_per_month_breaks, 
+     axes = F,
+     legend = NULL,
+     col = monthly_patterns_col,
+     alpha = 0.9,
+     mar = NA,
+     cex.main = 1.3)
+plot(southern_shp, add = T)
+
+plot(dry_days_per_month_r, 
+     "jan", 
+     main = "Jan",
+     type = "interval", 
+     breaks = dry_days_per_month_breaks, 
+     axes = F,
+     legend = NULL,
+     col = monthly_patterns_col,
+     alpha = 0.9,
+     mar = NA,
+     cex.main = 1.3)
+plot(southern_shp, add = T)
+
+plot(dry_days_per_month_r, 
+     "feb", 
+     main = "Feb",
+     type = "interval", 
+     breaks = dry_days_per_month_breaks, 
+     axes = F,
+     legend = NULL,
+     col = monthly_patterns_col,
+     alpha = 0.9,
+     mar = NA,
+     cex.main = 1.3)
+plot(southern_shp, add = T)
+
+plot(dry_days_per_month_r, 
+     "mar", 
+     main = "Mar",
+     type = "interval", 
+     breaks = dry_days_per_month_breaks, 
+     axes = F,
+     legend = NULL,
+     col = monthly_patterns_col,
+     alpha = 0.9,
+     mar = NA,
+     cex.main = 1.3)
+plot(southern_shp, add = T)
+
+plot(dry_days_per_month_r, 
+     "apr", 
+     main = "Apr",
+     type = "interval", 
+     breaks = dry_days_per_month_breaks, 
+     axes = F,
+     legend = NULL,
+     col = monthly_patterns_col,
+     alpha = 0.9,
+     mar = NA,
+     cex.main = 1.3)
+plot(southern_shp, add = T)
+```
+
+
+## On Dry Periods
+Dry days could be interspersed between rainy days, or grouped into streaks of consecutive dry days. A period of dryness could also be defined by the low cumulative rainfall it has received (regardless of the rainy status of individual days composing the period.) Both approaches are examined in this section.
+
+Reminder that a dry day can receive up to 3mm of rain while the definition of dry spell used for Anticipatory Action was a 14-day streak with no more than 2mm in total.
+
+### Streaks of Dry Days
+Streaks of dry days occurred throughout the region. The longest streak of consecutive dry days during the entire rainy season is represented below. The longest streaks lasted between `r minmax(static_r$max_streak)[1]` and `r minmax(static_r$max_streak)[2]` days.
+
+
+```{r longest-streak-plot, fig.show="hold", out.width="100%"}
+max_streak_breaks <- c(6, 8, 10, 12, 14, 16, 18, 20, 22) # upper value included in range and not lower value (6-8] = 6 excl and 8 included
+max_streak_labels <- c("7-8", "9-10", "11-12", "13-14", "15-16", "17-18", "19-20", "21-22")
+
+max_max_streak <- minmax(static_r$max_streak)[2]
+
+plot(static_r, 
```

Comment:
Hmm, I think these are calculated just across the rainy seasons for the areas, which makes sense. I just wonder if value in also looking at it in general, since it again looks like northern area is better even though it might be due to shorter rainy season.